cairo_zero/kakarot/library.cairo

```diff
@@ -410,7 +410,7 @@ namespace Kakarot {
         let (access_list) = alloc();
 
         return eth_call(
-            0, l1_sender, to, 2100000000, 1, value_u256, data_len, data, 0, access_list
+            0, l1_sender, to, 2100000000, 0, value_u256, data_len, data, 0, access_list
```

Comment:
wondering if we should create a contant called `INFINITE_GAS` or something explicit meaning : we don't want this call to fail because oog, this tx is paid for in the messaging flow by starknet and should not revert. Price is 0 so no refund will occur

Reply:
I agree 
Done with [bcc4d01](https://github.com/kkrt-labs/kakarot/pull/1584/commits/bcc4d01268ef4f7a2e0ee58f74ed571700913a4d)

Added a docstring on the function to explain why those values are choosen